### PR TITLE
Added script to scan repo for full essay images that have been

### PR DESCRIPTION
**Pull request by cryptobellum**
…inadvertently uploaded

# Description

Replace with description of PR

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [X] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

## Has This Been Tested

- [X] Yes
- [ ] No
- [ ] Not necessary

## Checklist

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [X] There are no merge conflicts

https://www.loom.com/share/a51bd6cc688241b888e431d347c6964f

### CODE REVIEW

File: utils/essaycheck_script.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
+from pathlib import Path
+import os
+
+"""
+This is a script that scans the repo for full essay images. Place this script in the top-level directory and run it from the command line.
```
Comment (BritVandy) [inline utils/essaycheck_script.py:5]: Thank you for all the documentation!

**Review by BritVandy — COMMENTED** [review]
The code is easy to understand and well documented. It makes sense why this was added. I'm not sure how the script works without a callable function, but other than that everything looks good.

**Review by Veilios — APPROVED** [review]
Clean Code all around! Looks great!

**Comment by cryptobellum** [comment]
@BritVandy After placing the script in the top level directory, cd into that directory and run python essaycheck_script.py from the command line.

**Comment by abdishifa234** [comment]
I have also the same problem when I tried to push the cleaned and transcribed data to the main repo and what happened is naming for transcribed and cleaned version/.png and gt.txt/ repeated again and again and solved by running such kind of scripts. You described everything in the loom video and if you can write some description or explanatory statement in your repo would a plus.

**Review by BritVandy — APPROVED** [review]
@cryptobellum That makes sense now. I definitely read that in the documentation. Not sure why I didn't understand until you explained again, but thank you!

**Review by abdishifa234 — APPROVED** [review]
I have also the same problem when I tried to push the cleaned and transcribed data to the main repo and what happened is naming for transcribed and cleaned version/.png and gt.txt/ repeated again and again and solved by running such kind of scripts. You described everything in the loom video and if you can write some description or explanatory statement in your repo would a plus.

**Pull request merged**